fastf1/core.py

```diff
@@ -3663,7 +3663,8 @@ class SessionResults(BaseDataFrame):
         - ``Position`` | :class:`float` |
           The drivers finishing position (values only given if session is
           'Race', 'Qualifying', 'Sprint Shootout', 'Sprint', or
-          'Sprint Qualifying').
+          'Sprint Qualifying'). This is the positions after any penalty and
+          disqualification is applied, not the race finishing position.
```

Comment:
This is actually not true for Sprint Qualifying, as that isn't supported by Jolpica-F1 (Ergast). In that case, the results are calculated from timing data and don't include any stewarding decisions.